src/utilities/texture_atlas.jl

```diff
@@ -263,6 +263,8 @@ function render(atlas::TextureAtlas, glyph::Char, font, downsample=5, pad=6)
     # the target pixel size of our distance field
     pixelsize = PIXELSIZE_IN_ATLAS[]
     # we render the font `downsample` sizes times bigger
+    # Make sure the font doesn't have a nutated font matrix from e.g. Cairo
```

Comment:
```suggestion
    # Make sure the font doesn't have a mutated font matrix from e.g. Cairo
```

Sorry for the not particularly insightful review and thanks a lot for fixing this!